### PR TITLE
homework09.Part2T28Generic

### PR DESCRIPTION
**Pull request by userhost1**
Generic

### CODE REVIEW

File: src/homework09/Part2T28Generic.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+//  Написать класс, который умеет хранить в себе массив любых типов данных (int, long etc.).
+//  Реализовать метод get(int index), который возвращает любой элемент массива по индексу.
+
+import java.util.ArrayList;
+
+public class Part2T28Generic {
+    public static void main(String[] args) {
+
+        ElementsClass<Object> elements = new ElementsClass<>();
```
Comment (TheFallen058) [inline src/homework09/Part2T28Generic.java]: Idea of task was to create class which can handle different types not in one object but in different.
I mean ElementsClass<Integer>, ElementsClass<String> and so on

File: src/homework09/Part2T28Generic.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    public static class ElementsClass<T> {
+        private ArrayList<T> elements = new ArrayList<T>();
+
+        public void add(T element) {
+            elements.add(element);
+        }
+
+        public T get(int index) {
```
Comment (TheFallen058) [inline src/homework09/Part2T28Generic.java:42]: Nice that you tried to handle exception, better to user if and check if elements are not null and elements size is bigger then required index

**Review by TheFallen058 — CHANGES_REQUESTED** [review]
Please change one task

File: src/homework09/Part2T28Generic.java (hunk 1 above)
Comment (userhost1) [reply src/homework09/Part2T28Generic.java]: possibly fixed. check please

File: src/homework09/Part2T28Generic.java (hunk 2 above)
Comment (userhost1) [reply src/homework09/Part2T28Generic.java:42]: fixed

**Comment by userhost1** [comment]
check please

File: src/homework09/Part2T28Generic.java (hunk 2 above)
Comment (userhost1) [reply src/homework09/Part2T28Generic.java:42]: > Nice that you tried to handle exception, better to user if and check if elements are not null and elements size is bigger then required index

fixed

File: src/homework09/Part2T28Generic.java (hunk 1 above)
Comment (userhost1) [reply src/homework09/Part2T28Generic.java]: > Idea of task was to create class which can handle different types not in one object but in different.
> I mean ElementsClass, ElementsClass and so on

fixed

**Review by TheFallen058 — APPROVED** [review]
Excellent

**Pull request merged**